test/integ/zeroDimBuffer/src/zeroDimBuffer.cpp

```diff
@@ -0,0 +1,137 @@
+/* Copyright 2021 Andrea Bocci
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+// make sure the CPU_B_SEQ_T_SEQ backend is always available
+#ifndef ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED
+#    define ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED
+#endif // ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED
+#include <alpaka/alpaka.hpp>
+#include <alpaka/test/acc/TestAccs.hpp>
+
+#include <catch2/catch.hpp>
+
+#include <cstdint>
+#include <iostream>
+
+constexpr int memset_value(int c)
+{
+    int v = c & 0xFF;
+    return (v | v << 8 | v << 16 | v << 24);
+}
+
+// 0- and 1- dimensional space
+using Idx = std::size_t;
+using Dim0D = alpaka::DimInt<0u>;
+using Dim1D = alpaka::DimInt<1u>;
+using Scalar = alpaka::Vec<Dim0D, Idx>;
+
+// enabled accelerators with 1-dimensional kernel space
+using TestAccs = alpaka::test::EnabledAccs<Dim1D, Idx>;
+
+TEMPLATE_LIST_TEST_CASE("zeroDimBuffer", "[zeroDimBuffer]", TestAccs)
+{
+    using DeviceAcc = TestType;
+    using Device = alpaka::Dev<DeviceAcc>;
+    using DeviceQueue = alpaka::Queue<DeviceAcc, alpaka::NonBlocking>;
+
+    using HostAcc = alpaka::AccCpuSerial<Dim1D, Idx>;
+    using Host = alpaka::DevCpu;
+    using HostQueue = alpaka::Queue<HostAcc, alpaka::Blocking>;
+
+    // check that a Scalar extent has exactly 1 element
+    Scalar scalar;
+    INFO("Scalar size: " << scalar.prod())
```

Comment:
Do we need such text output? I would rather replace it by a `CHECK(scalar.prod() == 1)`.

Reply:
Yes, I prefer to have it.

Reply:
Change to `CHECK`, assuming people don't look at the messages anyway.